test/fixtures/test-timer-disarm.js

```diff
@@ -0,0 +1,24 @@
+'use strict';                    
+require('../common');                              
+var assert = require('assert');                    
+var T = require('timers');                         
+var nbIntervalFired1 = 0;                            
+var nbIntervalFired2 = 0;                            
+var nbIntervalFired3 = 0;                            
+var timer1 = setInterval(function() {                
+  nbIntervalFired1++;                                
+  clearInterval(timer1);                             
+}, 11);                                              
+var timer2 = setInterval(function() {                
+  nbIntervalFired2++;                                
+  timer2.close();                                    
+}, 12);                                              
+var timer3 = setInterval(function() {                
+  nbIntervalFired3++;                                
+  T.unenroll(timer3);                                
+}, 13);                                              
+setTimeout(function onTimeout() {                    
+  assert.strictEqual(nbIntervalFired1, 1);           
+  assert.strictEqual(nbIntervalFired2, 1);           
+  assert.strictEqual(nbIntervalFired3, 1);           
+}, 100);
```

Comment:
Also, if you'd like to use `const` and arrow functions where possible, that is cool and generally we encourage it.